File: 11. Butterfly Pattern With Single Joint/CodeXam.cpp
#include <iostream>
using namespace std;
int main()
{
    for(int i=-3;i<=3;i++)
    {
        for(int j=1;j<=4-abs(i);j++)
        {
            cout<<"*"<<" ";
        }
        for(int j=1;j<=2*abs(i);j++)
        {
            cout<<" "<<" ";
        }
        for(int j=1;j<=4-abs(i);j++)
        {
            cout<<"*"<<" ";
        }
        cout<<endl;
    }
    return 0;
}

File: 11. Butterfly Pattern With Single Joint/CodeXam.java
public class CodeXam
{
    public static void main(String[] args)
    {
        for(int i=-3;i<=3;i++)
        {
            for(int j=1;j<=4-Math.abs(i);j++)
            {
                System.out.print("*" + " ");
            }
            for(int j=1;j<=2*Math.abs(i);j++)
            {
                System.out.print(" " + " " );
            }
            for(int j=1;j<=4-Math.abs(i);j++)
            {
                System.out.print("*" + " ");
            }
            System.out.println();
        }
    }
}
